### PR TITLE
link error line 20

### PR DESCRIPTION
**Pull request by dicktonyboy**
https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/ttn.html

the  link to the 880 installation goes to the kerlink installation and not to where it should go here:-    https://github.com/TheThingsNetwork/packet_forwarder/blob/master/docs/INSTALL_INSTRUCTIONS/IMST_RPI.md.

### CODE REVIEW

File: _content/gateways/packet-forwarder/ttn.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ The TTN Packet Forwarder is a new packet forwarder, developed by the Things Netw
 
 Currently, we have installation guides available to install the packet forwarder for [Multitech Conduit](https://github.com/TheThingsNetwork/packet_forwarder/blob/master/docs/INSTALL_INSTRUCTIONS/MULTITECH.md), [Kerlink IoT Station](https://github.com/TheThingsNetwork/packet_forwarder/blob/master/docs/INSTALL_INSTRUCTIONS/KERLINK.md) and [Raspberry Pi + iC880a](https://github.com/TheThingsNetwork/packet_forwarder/blob/master/docs/INSTALL_INSTRUCTIONS/KERLINK.md) setups.
```
Comment (jpmeijers) [inline _content/gateways/packet-forwarder/ttn.md]: Can you perhaps replace the incorrect URL with the correct one, and not add any unnecessary changes. The way it is now requires someone else to again modify the file and not just accept your changes.

Comment (dicktonyboy) [reply _content/gateways/packet-forwarder/ttn.md]: sorry - new to this format
